dev/pyRevitLabs.PyRevit.Runtime/ScriptConsole.cs

```diff
@@ -901,8 +920,29 @@ private string SaveContentsToTemp() {
             return tempHtml;
         }
 
-        private void OpenButton_Click(object sender, RoutedEventArgs e) {
-            Process.Start(string.Format("file:///{0}", SaveContentsToTemp()));
+        private void OpenButton_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                var filePath = SaveContentsToTemp();
+        
+                // Adjust for proper URI format
```

Comment:
can we get rid of the comments and white lines? this is a trivial method that is quite self explanatory